platform.in

```diff
@@ -81,7 +81,7 @@
 %__brp_strip %{_rpmconfigdir}/brp-strip %{__strip}
 %__brp_strip_comment_note %{_rpmconfigdir}/brp-strip-comment-note %{__strip} %{__objdump}
 %__brp_strip_shared %{_rpmconfigdir}/brp-strip-shared
-%__brp_strip_static_archive %{_rpmconfigdir}/brp-strip-static-archive %{__strip}
+%__brp_strip_static_archive %{_rpmconfigdir}/brp-strip-static-archive %{__strip} %{_smp_build_ncpus}
```

Comment:
I'd far prefer passing this in the environment along with RPM_BUILD_ROOT and all that to everything we process. All brp-* scripts are potential targets for parallelization, and passing the number of cpus via cli gets ugly real fast as every script takes different arguments in different order.